src/test/java/gov/healthit/chpl/aqa/stepDefinitions/Base.java

```diff
@@ -61,4 +61,28 @@ public WebDriverWait getWait() {
     public void setWait(final WebDriverWait wait) {
         this.wait = wait;
     }
+
+    /**
+     * Check whether the file is downloaded.
```

Comment:
This is not a _complicated_ method, but the documentation is incomplete. It not only checks whether the file is downloaded, it waits (forever) until that file exists. As well as updating the documentation, you should probably also put some sort of "it's been three hours of waiting, it's never going to happen, throw an exception" check in here (probably with less than three hours)